.flowconfig

```diff
@@ -22,6 +22,9 @@
 ; Ignore polyfills
 .*/Libraries/polyfills/.*
 
+; Ignore metro
+.*/node_modules/metro/.*
```

Comment:
we probably want to update this as well: https://github.com/facebook/react-native/blob/master/local-cli/templates/HelloWorld/_flowconfig

Reply:
I think that template is due to be removed anyway.

Reply:
Here's the PR that will move the template out: #16579

Reply:
Nevermind, this is a separate template (HelloNavigation). Taking a look.